src/tests/functional/shared_test_classes/include/shared_test_classes/base/low_precision_transformations/layer_transformation.hpp

```diff
@@ -30,35 +31,39 @@ class LayerTransformationParamsNGraphFactory {
 class LayerTransformationParamsFactory : public LayerTransformationParamsNGraphFactory {
 };
 
-class LayerTransformation : virtual public LayerTestsUtils::LayerTestsCommon {
+class LayerTransformation : virtual public ov::test::SubgraphBaseTest {
 protected:
     LayerTransformation();
 
-    static InferenceEngine::Blob::Ptr GenerateInput(
-        const ngraph::element::Type precision,
-        const InferenceEngine::TensorDesc& tensorDesc,
-        const float k = 1.f);
+    static std::pair<float, float> get_quantization_interval(ov::element::Type precision);
 
-    static std::pair<float, float> getQuantizationInterval(const ngraph::element::Type precision);
+    static std::string to_string(const ov::pass::low_precision::LayerTransformation::Params& params);
```

Comment:
Minor: so strange naming... What does it mean?

Reply:
This looks strange in diff. If you open file you can see that getQuantizationInterval renamed to get_quantization_interval